### PR TITLE
Distinguish between email and phone mask dashboard

### PR DESCRIPTION
**Pull request by Vinnl**
Previously, the email mask dashboard would still be called "Dashboard", whereas the phone masking dashboard was called "Phone Number". This was an artefact of us only having email masking before.

This PR fixes MPP-2549.

How to test: view the menu, both in small and wide viewports. It should say "Email Masks" and "Phone Masks".

# Checklist (Definition of Done)
- [x] Product Owner accepted the User Story (demo of functionality completed) or waived the privilege.
- [x] All acceptance criteria are met.
- [x] Jira ticket has been updated (if needed) to match changes made during the development process.
- [x] I've added or updated relevant docs in the docs/ directory
- [x] Jira ticket has been updated (if needed) with suggestions for QA when this PR is deployed to stage.
- [x] All UI revisions follow the [coding standards](https://github.com/mozilla/fx-private-relay/blob/main/docs/coding-standards.md), and use Protocol tokens where applicable (see `/frontend/src/styles/tokens.scss`).
- [x] Commits in this PR are minimal and [have descriptive commit messages](https://chris.beams.io/posts/git-commit/).
- [x] l10n changes have been submitted to the l10n repository, if any. https://github.com/mozilla-l10n/fx-private-relay-l10n/pull/117

### CODE REVIEW

File: frontend/src/components/layout/navigation/DashboardSwitcher.tsx [hunk 1]
```diff
@@ -18,15 +18,15 @@ export const DashboardSwitcher = () => {
       {/* Email Mask Btn */}
       <Link href="/accounts/profile">
         <a
-          className={`${styles["nav-mask-phone-icon"]} ${
+          className={`${styles["nav-mask-email-icon"]} ${
```
Comment (Vinnl) [inline frontend/src/components/layout/navigation/DashboardSwitcher.tsx:21]: This was just an inconsistency — this is the dashboard switcher's email mask link, but it was referring to phone masks.

image

File: frontend/src/components/layout/navigation/Navigation.module.scss [hunk 2]
```diff
@@ -24,7 +24,6 @@
   color: inherit;
 
   &.is-active {
-    padding-top: $spacing-xs;
```
Comment (Vinnl) [inline frontend/src/components/layout/navigation/Navigation.module.scss:27]: The currently-active link was slightly off-centre due to this padding, which was more noticeable now that the strings are longer.

File: frontend/src/components/layout/navigation/Navigation.tsx [hunk 3]
```diff
@@ -38,9 +38,32 @@ export const Navigation = (props: Props) => {
...
+      <a
+        className={`${styles.link} ${styles["home-link"]} 
+          ${styles["hidden-mobile"]} 
+          ${router.pathname === "/" ? styles["is-active"] : ""}`}
+      >
+        {l10n.getString("nav-home")}
+      </a>
+    </Link>
+  );
+
```
Comment (lloan) [inline frontend/src/components/layout/navigation/Navigation.tsx:66]: Question (non-blocking): Should this still be called a `homeLink`. Couple of ways to approach this - one being that we render conditionally by checking if user is logged in or we just change the name of the variable. Or we don't do anything and just add a comment. Not important at the moment, but did want to make sure to call it out.

**Review by lloan — APPROVED** [review]
Tested and it looks good! 

Do we have any tests we need to update based on this?

File: frontend/src/components/layout/navigation/Navigation.tsx (hunk 3 above)
Comment (Vinnl) [reply frontend/src/components/layout/navigation/Navigation.tsx]: Oh! That's a pattern I haven't thought of before, but I like it! Saves a bunch of nesting. Is 5d6fdb96 what you were thinking of?

Tests are a good idea as well, added some in e7eaa30c.

**Comment by Vinnl** [comment]
Since it's been open for a while with an initial approval, I'm going to assume this is good to go :)

**Pull request merged**